source/manual/conventions-for-rails-applications.html.md

```diff
@@ -65,9 +65,7 @@ Ruby / Rails applications:
 - [plek][] - Utility tool to access determine base URLs for GOV.UK applications
   for the current environment
 - [rubocop-govuk][] - Provides GOV.UK linting rules for Ruby, Rails, RSpec and
-  Rake
-- [scss-lint-govuk][] - Provides linting for SCSS files according to GDS
-  conventions
+  Rake (we have further documentation on [configuring linting][configure-linting])
```

Comment:
How about making this a new one-liner section - about linting? I don't see much point tacking it onto an already long line, as it'll just get missed. It's also only tangentially related to rake.

Reply:
Sure, I was a bit lazy with this. I've updated this now to have a short section.